.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: This template is for reporting bugs you have found.
+title: "[DATE]: [BUG TITLE]"
+labels: Needs Triage
+assignees: 
+---
+
+## Information
+
+Device:
+
+Android version:
+
+App Version:
```

Comment:
I think this doesn't really apply for this repo.

Reply:
Oh, I see the issue :) I'll fix it